wdl-lint/tests/lints/snake-case/source.wdl

```diff
@@ -0,0 +1,33 @@
+# Test SnakeCase rule
+
+version 1.0
+
+workflow BadWorkflow {
+    call BadTask
+    call good_task
+}
+
+task BadTask {
+    command <<<
+        echo "Hello World"
+    >>>
+    output {
+        File badOut = "out.txt"
+    }
```

Comment:
Relating to the above comments, we should probably add `input` sections that have both bound and unbound decls.